microsoft-edge/webview2/how-to/publish-uwp-app-store.md

```diff
@@ -44,9 +44,11 @@ The Microsoft Store requires submitting a packaged app.  Package your app as fol
 
 1. Associate the application with the reserved name, by selecting **Project** > **Publish** > **Associate App with the Store**.
 
-1. Create the app package, by selecting **Project** > **Publish** > **Create App Packages**.  Select the target configurations.  Only 64-bit versions of the WebView2 Runtime will work in UWP:
+1. Create the app package, by selecting **Project** > **Publish** > **Create App Packages**.  Select the target configurations:
 
-   ![Package Selection](publish-uwp-app-store-images/package-selection.png)
+   ![Selecting target configurations](publish-uwp-app-store-images/package-selection.png)
+
+   32-bit and 64-bit versions of the WebView2 Runtime will work in UWP on non-ARM platforms.  32-bit is not supported for ARM, due to dependencies on Microsoft Edge, which doesn't support ARM32.
```

Comment:
32-bit and 64-bit versions of the WebView2 Runtime will work in UWP on non-ARM platforms.  32-bit is not supported for ARM, due to dependencies on Microsoft Edge.